### PR TITLE
Add preliminary schemas for VPR XML files.

### PR DESCRIPTION
**Pull request by Laksen**
This could be used for validation and as a potential base for autogenerating import/export code detailed in https://github.com/SymbiFlow/ideas/issues/4

### CODE REVIEW

File: common/xml/fpga_architecture.xsd [hunk 1]
```diff
@@ -0,0 +1,667 @@
+<?xml version="1.0"?>
```
Comment (mithro) [inline common/xml/fpga_architecture.xsd]: Would you mind adding a comment which links to the VPR documentation for this file?

File: common/xml/packed_netlist.xsd [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
```
Comment (mithro) [inline common/xml/packed_netlist.xsd]: Would you mind adding a comment which links to the VPR documentation for this file?

File: common/xml/routing_resource.xsd [hunk 3]
```diff
@@ -0,0 +1,240 @@
+<?xml version="1.0"?>
```
Comment (mithro) [inline common/xml/routing_resource.xsd]: Would you mind adding a comment which links to the VPR documentation for this file?

**Comment by Laksen** [comment]
Added that

To validate you can use `xmllint --noout --schema schema.xsd file.xml`

**Comment by mithro** [comment]
Okay this all looks good! Going to merge and will try and do some testing later today.

**Pull request merged**